libkbfs/folder_branch_ops.go

```diff
@@ -3960,6 +3950,36 @@ func (fbo *folderBranchOps) notifyOneOpLocked(ctx context.Context,
 	afterUpdateFn func() error) error {
 	fbo.headLock.AssertLocked(lState)
 
+	// We need to get unlinkPath before calling UpdatePointers so that
+	// nodeCache.Unlink can properly update cachedPath.
+	unlinkPath, toUnlink := func() (path, bool) {
+		var (
```

Comment:
No benefit performance wise; I just thought it looked clearer to group them together. But I've no problem with not using it as well; will change